.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2.1
+jobs:
+
+  semgrep:
+    docker:
+      - image: returntocorp/semgrep:develop
+    working_directory: /src
+    steps:
+      - checkout
+      - run: semgrep --config semgrep-core/semgrep.yml --error --strict --exclude ocaml --exclude TODO --exclude _build semgrep-core
```

Comment:
Can we store this configuration in the cloud app instead of locally? Even better to dogfood two things at once.

Reply:
We already dogfood the cloud config in semgrep-app. Here I want to remove extra layers,
and it's a good way to dogfood our circleCI support :)